src/ioctls/vcpu.rs

```diff
@@ -961,7 +961,9 @@ mod tests {
             userspace_addr: load_addr as u64,
             flags: KVM_MEM_LOG_DIRTY_PAGES,
         };
-        vm.set_user_memory_region(mem_region).unwrap();
+        unsafe {
```

Comment:
It's nice to have a comment outside the `unsafe` block explaining why the code is actually safe to use. This seems to be pretty pervasive around here :(

Reply:
Not in the tests though. But if you want to have it in the tests as well, I can add a comment.